gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/internal/model/ApiKeyMongo.java

```diff
@@ -31,6 +33,8 @@
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
+@ToString(of = { "key", "expireAt", "revoked", "revokedAt" })
+@EqualsAndHashCode(of = { "id" })
```

Comment:
The same with the @EqulasAndHashCode actually: 

https://projectlombok.org/features/EqualsAndHashCode
<img width="732" alt="image" src="https://github.com/user-attachments/assets/4b2a751d-5aae-43bd-8ab4-3f1e9c85eacb">

Reply:
Oh I don't know it, it's fixed